tools/drdeploy.c

```diff
@@ -318,6 +318,7 @@ const char *options_list_str =
 #    endif
     "       -use_dll <dll>     Inject given dll instead of configured DR dll.\n"
     "       -use_alt_dll <dll> Use the given dll as the alternate-bitwidth DR dll.\n"
+    "       -toolsdir <dir>    Directory containing tool configuration files.\n"
```

Comment:
Hmm, is "-tooldir" or "-tool_dir" better?  Subjective I guess; singular ("tool" instead of "tools") sounds better to me b/c of English grammar preferring singular in compounds (e.g., "dogwalker" not "dogswalker") but if you strongly prefer "-toolsdir" go with that.